src/main/scala/beam/agentsim/agents/parking/ChoosesParking.scala

```diff
@@ -66,7 +67,8 @@ trait ChoosesParking extends {
         this.currentTourBeamVehicle,
         remainingTripData,
         attributes.valueOfTime,
-        parkingDuration
+        parkingDuration,
+        currentDriverId = id.asInstanceOf[Id[Person]].toString
```

Comment:
what does the `.asInstanceOf[Id[Person]]` give you here? i think you can probably remove it.